suites/reef/cephfs/tier-2_cephfs_test-nfs_ha.yaml

```diff
@@ -155,3 +155,9 @@ tests:
       desc: nfs-ha-node-replacement
       polarion-id: CEPH-83575096
       abort-on-fail: false
+  - test:
+      name: nfs-ha-snapshot-lifecycle
+      module: cephfs_nfs.test_cephfs_nfs_ha_snapshot_lifecycle.py
```

Comment:
As we are not performing any snapshot lifecycle on the NFS exports, let's rename it to validate_snaps_after_failover or something similar.

We can use the name snapshot_lifecycle when the support for a snapshot on nfs exports are available in future releases